src/Node/Globals.js

```diff
@@ -1,11 +1,11 @@
 "use strict";
 
-exports.__dirname = __dirname;
-exports.__filename = __filename;
-exports.unsafeRequire = require;
+export {__dirname};
+export {__filename};
```

Comment:
These don't exist in ES modules. Rather, one must use `import.meta.url` and some other stuff to get the same effect.